src/edges/edges.js

```diff
@@ -22,27 +22,29 @@ const getInterests = (interests) => {
 
 const getStroke = (shareValues) => {
   const { exact, minimum, maximum } = shareValues || {};
-  if (exact === undefined) {
+  if (exact === undefined && minimum <= maximum) {
     if (minimum !== undefined && maximum !== undefined) {
       return (minimum + maximum) / 2 / 10;
     } else {
       return DEFAULT_STROKE;
     }
   } else {
-    return exact > 0 ? exact / 10 : 1;
+    return exact / 10;
   }
 };
 
 const getText = (shareValues, type) => {
   const { exact, minimum, maximum } = shareValues || {};
-  if (exact === undefined) {
+  if (exact === undefined && minimum < maximum) {
     if (minimum !== undefined && maximum !== undefined) {
       return `${type} ${minimum} - ${maximum}%`;
     } else {
       return ``;
     }
-  } else {
+  } else if (exact !== undefined) {
     return `${type} ${exact}%`;
+  } else {
+    return `${type} ${minimum}%`;
```

Comment:
Here I think minimum is returned even when minimum > maximum, which I don't think is desirable